data/json/recipes/food/offal_dishes.json

```diff
@@ -290,5 +290,85 @@
     ],
     "charges": 8,
     "autolearn": true
+  },
+  {
+    "type": "recipe",
+    "result": "lung_provence",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 4,
+    "time": "55 m",
+    "batch_time_factors": [ 83, 5 ],
+    "charges": 4,
+    "book_learn": [ [ "offalcooking", 2 ] ],
+    "qualities": [ { "id": "COOK", "level": 3 }, { "id": "CUT", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 50, "LIST" ] ] ],
+    "components": [
+      [ [ "lung", 4 ] ],
+      [ [ "flour", 4 ] ],
+      [ [ "onion", 1 ], [ "irradiated_onion", 1 ] ],
+      [ [ "cooking_oil", 1 ], [ "cooking_oil2", 1 ] ],
+      [ [ "salt", 6 ] ],
+      [ [ "pepper", 6 ] ],
+      [ [ "worthy_wine", 3 ] ],
+      [ [ "garlic_clove", 1 ] ],
+      [ [ "sauce_pesto", 1 ], [ "sauce_red", 1 ], [ "tomato", 1 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "tÖttchen",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 4,
+    "time": "35 m",
+    "batch_time_factors": [ 83, 5 ],
+    "charges": 8,
+    "book_learn": [ [ "offalcooking", 2 ] ],
+    "qualities": [ { "id": "COOK", "level": 3 }, { "id": "CUT", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 50, "LIST" ] ] ],
+    "components": [
+      [ [ "lung", 4 ] ],
+      [ [ "kidney", 4 ] ],
+      [ [ "brain", 4 ] ],
+      [ [ "flour", 12 ] ],
+      [ [ "onion", 3 ], [ "irradiated_onion", 3 ] ],
+      [
+        [ "any_butter", 4 ] ],
```

Comment:
```suggestion
        [ [ "any_butter", 4 ] ],
```

Reply:
That shows a new error now there,  is it because I'm using a requirement list?

Reply:
Yes, you need `"LIST"` as the third element in the array if you're using a requirement.